tests/quantization/hqq/test_hqq.py

```diff
@@ -109,7 +109,7 @@ def test_fp16_quantized_model(self):
         Simple LLM model testing fp16
         """
         compute_dtype = torch.float16
-        device = "cuda:0"
+        torch_device = "cuda:0"
```

Comment:
sorry, I wasn't clear. I didn't mean that we should rename the variable torch_device. I meant we should take the variable torch_device from transformers.testing_utils, as this will be more flexible across different setups and hardware e.g. [like here for aqlm](https://github.com/huggingface/transformers/blob/5cf3e6bf05264726ffa61709787312b09cb4973e/tests/quantization/aqlm_integration/test_aqlm.py#L30)

Reply:
https://github.com/huggingface/transformers/pull/29637/commits/ca07f5a3e56ea1af6d695a13dbb4fa45fbbff750